model-optimizer/extensions/front/tf/RetinaNetFilteredDetectionsReplacement.py

```diff
@@ -260,4 +260,17 @@ def generate_sub_graph(self, graph: Graph, match: SubgraphMatch):
             dict(name=detection_output_op.attrs['type'], nms_threshold=iou_threshold, clip_after_nms=1, normalized=1,
                  variance_encoded_in_target=0, background_label_id=1000))
 
+        # As outputs are replaced with a postprocessing node, outgoing tensor names are no longer
+        # correspond to original tensors and should be removed from output->Result edges
+        for out_name in match.custom_replacement_desc.instances['end_points']:
+            out_node = Node(graph, out_name)
+            for out_idx in out_node.out_nodes():
+                result_node = out_node.out_node(out_idx)
+                fw_info_list = get_edge_attribute_between_nodes(out_node, result_node, 'fw_tensor_debug_info')
+                new_fw_info = []
+                for fw_info in fw_info_list:
+                    if fw_info is not None and len(fw_info) >= 2:
+                        new_fw_info.append((fw_info[0], fw_info[1], None))
+                set_edge_attribute_between_nodes(out_node, result_node, 'fw_tensor_debug_info', new_fw_info)
```

Comment:
I propose to create a separate function for this procedure since it is used in a couple of transformations